terratest/test/helper.go

```diff
@@ -19,25 +19,51 @@ package test
 
 import (
 	"fmt"
-	"strings"
-	"sort"
 	"testing"
-	"github.com/gruntwork-io/terratest/modules/shell"
+	"time"
+
+	"github.com/miekg/dns"
```

Comment:
https://github.com/AbsaOSS/gopkg/blob/main/dns/dns.go uses https://github.com/lixiangzhong/dnsutil as dig abstraction layer, it is built on top of github.com/miekg/dns too. Maybe it makes sense to use github.com/miekg/dns  too?

Reply:
makes sense, I'll amend once I realize why terratest fails to query CoreDNS in GHA

Reply:
@somaritane I can use following package. DNS Port is hardcoded there https://github.com/lixiangzhong/dnsutil/blob/master/dig.go#L245

Reply:
@k0da you mean you can't, yep I see, might require changes in upstream, sigh.

Reply:
We have e2e dig function here https://github.com/AbsaOSS/k8gb/blob/master/terratest/test/helpers.go#L50

We can use it here or in a transitive e2e test on k8gb side.

Reply:
@ytsarev it was used here, prior this PR :)

Reply:
ah, right. So something to extend in k8gb test suite as it will be closer to end-user scenario